File: leetcode/array/RemoveDuplicatesFromSortedArray_26/Main.java
import java.util.Arrays;

public class Main {

    public static void main(String[] args) {
        Solution solution = new Solution();

        int[] inputArr = {0,0,1,1,1,2,2,3,3,4};
        int size = solution.removeDuplicates(inputArr);
        System.out.println(Arrays.toString(inputArr));
        System.out.println(size);
    }
}

File: leetCode/array/RemoveDuplicatesFromSortedArray_26/main.py
class Solution:
    def removeDuplicates(self, nums):
        k = 1

        for i in range(len(nums) - 1):
            if nums[i] != nums[i + 1]:
                nums[k] = nums[i + 1]
                k += 1

        return k

a = Solution()
a.removeDuplicates([1,1,2])
pass